### PR TITLE
fix: Weird redirect after removing RMRK

### LINKED ISSUE #11022: Weird redirect after removing RMRK
### Context 

I visited https://koda.art/**rmrk**/detail/9546415-b4d42c2aa369ee1f27-PREZT-THINKER-0000000000000118
and was redirected to https://koda.art/**ahp**/detail/9546415-b4d42c2aa369ee1f27-PREZT-THINKER-0000000000000118

### Task

1. when user visits /rmrk || /ksm on non-transfer route he/she should be redirected to /ahp landing

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #11022

### CODE REVIEW

**Comment by Jarsen136** [comment]
> atm you can open the explore dropdown and select , which redirects you to the landing of with `/ahp`
> 
> could you hide it using `explorerVisible`
> 

Updated

**Comment by hassnian** [comment]
> > atm you can open the explore dropdown and select , which redirects you to the landing of with `/ahp`
> > could you hide it using `explorerVisible`
> 
> Updated

forgot about the drops , can you also hide that :D


https://github.com/user-attachments/assets/357c8a0f-a1ac-4a15-b70d-6e2a7a84625d

**Comment by Jarsen136** [comment]
> > > atm you can open the explore dropdown and select , which redirects you to the landing of with `/ahp`
> > > could you hide it using `explorerVisible`
> > 
> > 
> > Updated
> 
> forgot about the drops , can you also hide that :D
> 

There is no need because it makes sense to show the entrance to drops page

**Comment by hassnian** [comment]
> > > > atm you can open the explore dropdown and select , which redirects you to the landing of with `/ahp`
> > > > could you hide it using `explorerVisible`
> > > 
> > > 
> > > Updated
> > 
> > 
> > forgot about the drops , can you also hide that :D
> 
> There is no need because it makes sense to show the entrance to drops page

i't hard to see in the video but I'm not going back manually I'm being redirected back to the transfer page, If I can click I should be able to go otherwise we should hide it

**Comment by Jarsen136** [comment]
> > > > > atm you can open the explore dropdown and select , which redirects you to the landing of with `/ahp`
> > > > > could you hide it using `explorerVisible`
> > > > 
> > > > 
> > > > Updated
> > > 
> > > 
> > > forgot about the drops , can you also hide that :D
> > 
> > 
> > There is no need because it makes sense to show the entrance to drops page
> 
> i't hard to see in the video but I'm not going back manually I'm being redirected back to the transfer page, If I can click I should be able to go otherwise we should hide it

It looks like a weird bug on this page. I just made a fix.

**Review by hassnian — APPROVED** [review]

File: middleware/redirects.global.ts [hunk 1]
```diff
@@ -31,6 +31,10 @@ export default defineNuxtRouteMiddleware((route) => {
         return accountId.value ? `/${urlPrefix.value}/u/${accountId.value}` : `/${urlPrefix.value}`
       },
     },
+    {
+      cond: (val: string) => ['ksm', 'rmrk', '/dot'].some(prefix => val.startsWith(`/${prefix}`) && !val.startsWith(`/${prefix}/transfer`)),
```
Comment (vikiival) [inline middleware/redirects.global.ts:35]:
```suggestion
      cond: (val: string) => ['ksm', 'rmrk', 'dot'].some(prefix => val.startsWith(`/${prefix}`) && !val.startsWith(`/${prefix}/transfer`)),
```

?

**Review by vikiival — COMMENTED** [review]
cc @Jarsen136 
Why is slash there ?

**Comment by Jarsen136** [comment]
> cc @Jarsen136 Why is slash there ?

mistake : )

**Review by vikiival — APPROVED** [review]

**Pull request merged**